src/content/guide/how-to-build-a-product/authentication.md

```diff
@@ -0,0 +1,423 @@
+Your web app will use the generated client ID and secret  when you hit the
+Particle API on behalf of your organization and your customers to manage all the
+devices in your product fleet.
+
+In the API reference, note how there is an optional parameter called
+`organization`. In order to create client credentials for your
+organization, you should pass the **slug** of your organization as this
+parameter. The easiest way to find the slug of your organization is in
+the URL address when viewing your organization's dashboard, just after
+the `/` in `dashboard.particle.io/`. Example:
+`dashboard.particle.io/my-org`, where `my-org` is the organization slug.
+The API request to create client credentials scoped to your organization
+will look like this:
+
+```bash
+$ curl -d name=MyApp -d type=installed -d organization:my-org -d access_token=1234
+https://api.particle.io/v1/clients
+```
+
+The `type` of the OAuth client will depend on your authentication method
+and application medium (web or mobile). More on this
+[later](#choosing-an-authentication-method).
+
+There is also an optional `scope` parameter, which can be set to
+`create_customer`. This option will scope the client to only be able
+to create customers for your organization. Pass this parameter if you
+are using [simple authentication](#simple-authentication).
+
+If successful, the API will return your client ID and secret in the
+response to the request. **Never expose your client credentials,
+especially if they are unscoped**. Credentials are sensitive pieces of
+information that, if exposed, could allow unauthorized people or
+applications to interact with your organization's data and devices. [More
+on registering clients](http://tools.ietf.org/html/rfc6749#section-2).
+
+## Access Tokens
+A related concept to understand is how Particle uses access
+tokens for authentication and security. If you have ever logged into the
+Web IDE, called a function on a Particle device, or read a variable via
+the API, you are already using access tokens! It is important to note
+that OAuth credentials are needed to create access tokens.
+
+### What's an access token?
+
+An *access token* is a special code that is tied to a Particle customer
+or user, that allows reading data from and sending commands to that
+person's device(s). Any API endpoint that returns private information,
+or allows control of an individual's device requires an access token.
+Tokens *inherit scopes* from the OAuth clients used to create them, and
+can have more specific scopes of their own. [More on access
+tokens](http://tools.ietf.org/html/rfc6749#section-1.4).
+
+Here's a specific example. Let's say that on one of your own personal
+devices (with device ID `0123456789abcdef01234567`), you want to call the `lightsOn` function. Here is how you would
+do this using the API:
+
+```bash
+curl
+https://api.particle.io/v1/devices/0123456789abcdef01234567/lightsOn -d
+arg=livingRoom -d access_token=1234
```

Comment:
Actually, throughout this guide, we should recommend that access tokens be sent in an `Authorization` header.

Also, let's avoid case sensitive URLs.

```
curl -H 'Authorization: Bearer 1234'
  -d arg=livingRoom -X POST
  https://api.particle.io/v1/devices/0123456789abcdef01234567/lights_on
```